### DISCUSSION TITLE
rate limit capture

### DISCUSSION THREAD

**Discussion by VinylVault** [joalla/discogs_client#25] [Ideas] [answered]
can we look at some kind of automatic rate limit detection and throttling for those endpoints that reuire it at the appropriate levels, as much as i believe that 60 requests per minute is far too low a value its what we have to play with, and it is very hard to access the rate limit information (especially for new python users - like me) having some sort of built in automated system would vastly improve the usability of the module, even make the option user disableable,  the default position being on

**Comment by alifhughes** [comment, marked as answer]
Hey @VinylVault this is a duplicate of [this question here](https://github.com/joalla/discogs_client/discussions/21) so I'm going to close this one. Please keep an eye on the other thread for updates. Thanks!
